### PR TITLE
python: fix python 3.8 Syntax Warning

### LINKED ISSUE #2230: pyinstaller shows SyntaxWarning: "is" with a literal. Did you mean "=="?
I am trying to package a plotly app with `pyinstaller`. During the process I get these warnings:

c:\users\soere\miniconda3\envs\ms-mint\lib\site-packages\_plotly_utils\utils.py:214: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if (iso_string.split("-")[:3] is "00:00") or (iso_string.split("+")[0] is "00:00"):
c:\users\soere\miniconda3\envs\ms-mint\lib\site-packages\plotly\figure_factory\_candlestick.py:194: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if direction is "increasing":
c:\users\soere\miniconda3\envs\ms-mint\lib\site-packages\plotly\figure_factory\_candlestick.py:199: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif direction is "decreasing":
c:\users\soere\miniconda3\envs\ms-mint\lib\site-packages\plotly\figure_factory\_ohlc.py:176: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if direction is "increasing":
c:\users\soere\miniconda3\envs\ms-mint\lib\site-packages\plotly\figure_factory\_ohlc.py:179: SyntaxWarning: "is" with a literal. Did you mean "=="?

### PR DESCRIPTION
**Pull request by sgn**
Closes #2230

### CODE REVIEW

File: packages/python/plotly/_plotly_utils/utils.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ class NotEncodable(Exception):
 def iso_to_plotly_time_string(iso_string):
     """Remove timezone info and replace 'T' delimeter with ' ' (ws)."""
     # make sure we don't send timezone info to plotly
-    if (iso_string.split("-")[:3] is "00:00") or (iso_string.split("+")[0] is "00:00"):
+    if (iso_string.split("-")[:3] == "00:00") or (iso_string.split("+")[0] == "01:00"):
```
Comment (nicolaskruchten) [inline packages/python/plotly/_plotly_utils/utils.py]: Thanks for this PR! Is there a reason the `00` turned into `01` here?

Comment (sgn) [reply packages/python/plotly/_plotly_utils/utils.py]: Sorry, it was a change that not intended, removed.

**Comment by nicolaskruchten** [comment]
This looks great, thank you! It will go out as part of the 4.5.4 release today :)

**Comment by lewi0332** [comment]
Hi all, 

I am also seeing this same issue in the chart-studio tools.py file.  Sorry if this isn't the right place, but I didn't see a chart studio specific issues spot. 

chart-studio==1.1.0.  

```
2021-05-12T14:51:57.985399+00:00 app[web.1]: /app/.heroku/python/lib/python3.9/site-packages/chart_studio/tools.py:290: SyntaxWarning: "is" with a literal. Did you mean "=="?
2021-05-12T14:51:57.985401+00:00 app[web.1]:   if share_key is "":
```


Additionally I'm not actually calling for Chart-studio to be installed, it is being added by a pip requirement of something else.

**Pull request merged**